src/v2/Apps/Fair/Components/FairBoothRail/FairBoothRail.tsx

```diff
@@ -43,14 +56,29 @@ export const FairBoothRail: React.FC<FairBoothRailProps> = ({
     action: ActionType.clickedArtworkGroup,
   }
 
+  if (show.href) {
+    const params = removeDefaultValues(filters!, {
+      defaultValues: initialBoothFilterState,
+    })
+    const preparedParams = paramsToSnakeCase(params)
+    const queryString = qs.stringify({
+      ...preparedParams,
+      focused_booths: true,
+    })
+    const backHref = `${match.location.pathname}?${queryString}`
+    const encodedBackHref = encodeURIComponent(backHref)
+
+    link = `${show.href}?back_to_fair_href=${encodedBackHref}`
+  }
```

Comment:
Initially, I thought to scroll the page to a specific item from which the user navigated to the Show page. But during testing, undesirable behavior was found that worsened the user's experience with the page.

The problem is that initially, when loading the page, instead of real images for the elements in the Booths section, we display loading placeholders of fixed size. When the actual image is rendered instead of loading placeholder, the final size of the element could change. For this reason, the scroll offset to a specific item in the Booths section, which was made when navigating to the Fair page from "Back to <Fair>" banner, could not correspond to the offset that will be after all the images for the elements are loaded.